### PR TITLE
opam 2.1.2 release

### PR DESCRIPTION
**Pull request by kit-ty-kate**

### CODE REVIEW

File: opam-2-1-2.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+authors: [
+  "Kate Deplaix - OCamlLabs" {"mailto:kit.ty.kate(à)disroot.org"}
+  "David Allsopp - OCamlLabs" {"mailto:David.Allsopp(à)cl.cam.ac.uk"}
+  "Raja Boujbel - OCamlPro" {"mailto:raja.boujbel(à)ocamlpro.com"}
+  "Louis Gesbert - OCamlPro" {"mailto:louis.gesbert(à)ocamlpro.com"}
+]
+date: "2021-12-08"
+--BODY--
+
+_Feedback on this post is welcomed on [Discuss](TODO)!_
```
Comment (rjbou) [inline opam-2-1-2.md:11]:
```suggestion
<!--
_Feedback on this post is welcomed on [Discuss](https://discuss.ocaml.org/t/XXXXXX)!_
-->
```

File: opam-2-1-2.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+title: "opam 2.1.2 release"
+authors: [
+  "Kate Deplaix - OCamlLabs" {"mailto:kit.ty.kate(à)disroot.org"}
+  "David Allsopp - OCamlLabs" {"mailto:David.Allsopp(à)cl.cam.ac.uk"}
+  "Raja Boujbel - OCamlPro" {"mailto:raja.boujbel(à)ocamlpro.com"}
+  "Louis Gesbert - OCamlPro" {"mailto:louis.gesbert(à)ocamlpro.com"}
```
Comment (rjbou) [inline opam-2-1-2.md:6]:
```suggestion
  "Kate Deplaix - OCamlLabs" {"mailto:kit.ty.kate(à)disroot.org"}
```
you're the only author of the blog post :)

File: opam-2-1-2.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+]
+date: "2021-12-08"
+--BODY--
+
+_Feedback on this post is welcomed on [Discuss](TODO)!_
+
+We are pleased to announce the minor release of [opam 2.1.2](https://github.com/ocaml/opam/releases/tag/2.1.2).
+
+This opam release consist of backported fixes, including:
```
Comment (rjbou) [inline opam-2-1-2.md:15]:
```suggestion
This opam release consist of [backported](https://github.com/ocaml/opam/issues/4920) fixes, including:
```

File: opam-2-1-2.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+* Fallback on dnf if yum does not exist on RHEL-based systems [#4825](https://github.com/ocaml/opam/pull/4825)
+* Use --no-depexts in CLI 2.0 mode [#4908](https://github.com/ocaml/opam/pull/4908)
```
Comment (dra27) [inline opam-2-1-2.md:20]: Formatting only:
```suggestion
* Fallback on `dnf` if `yum` does not exist on RHEL-based systems ([#4825](https://github.com/ocaml/opam/pull/4825))
* Use `--no-depexts` in CLI 2.0 mode ([#4908](https://github.com/ocaml/opam/pull/4908))
```

Comment (dra27) [reply opam-2-1-2.md:20]: Could possibly add "_in CLI 2.0 mode._ This further improves the use of opam 2.1 as a drop-in replacement for opam 2.0 in CI, for example with setup-ocaml in GitHub Actions."

**Review by dra27 — COMMENTED** [review]
Formatting and possible suggestion to expand on one of the fixes

File: opam-2-1-2.md (hunk 2 above)
Comment (kit-ty-kate) [reply opam-2-1-2.md:6]: I mean I’ve mostly copy/pasted the previous ones so depends on your definition of "author"

Comment (rjbou) [reply opam-2-1-2.md]: Usually, it's the writer of the blog post

**Pull request merged**